tensorflow/core/transforms/remapper/tests/fusedbatchnormEx_test.mlir

```diff
@@ -0,0 +1,24 @@
+// RUN: tfg-transforms-opt --pass-pipeline='tfg-shape-inference,tfg-remapper' %s | FileCheck %s
```

Comment:
could you please fix this
-pass-pipeline='builtin.module(tfg-shape-inference,tfg-remapper)'

that seems to be failing internally (same in the other .mlir test file)